.github/workflows/release-signoff-chrome.yml

```diff
@@ -34,6 +34,11 @@ jobs:
           cd opensearch-${{ env.VERSION }}/
           ./opensearch-tar-install.sh &
           timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:${{ env.OPENSEARCH_INITIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
+      - uses: derek-ho/start-opensearch@v3
```

Comment:
I don't know which files we actually need to add this, so i just added it to a few for now.

Reply:
if it will be used by all plugins, we could add to workflow template file `release-e2e-workflow-template.yml` and `release-e2e-workflow-template-windows.yml`

Reply:
Done.